src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -303,6 +304,11 @@ export default class ChangeBusinessType extends Mixins(CommonMixin, SharedMixin)
     return this.getResource.changeData?.typeChangeInfo
   }
 
+  /** v-if for tooltip (change firm and coop)  */
+  get showChangeInfoTooltip ():boolean {
+    return this.isChangeRegFiling || this.isEntityTypeFirm || this.isEntityTypeCP
```

Comment:
The first 2 conditions overlap: a ChangeReg filing can only be done by a firm.

I would keep the second condition since firms can also do Correction and Conversion filings.

Reply:
This was the existing condition in HTML moved added for cp and moved to getter. Sorry i didnt get this exactly something like this? return this.isChangeRegFiling || this.isEntityTypeFirm || this.isSpecialResolutionFiling   ?

Reply:
Yes, you updated an existing condition -- thank you.

I'm saying you don't need the first condition here because only firms can do a ChangeReg filing, so if the second condition is true then the first condition is also/always true.

Reply:
oh, I see, 
so we need only 
this.isEntityTypeFirm || this.isEntityTypeCP
or 
 this.isChangeRegFiling  || this.isSpecialResolutionFiling
?

Reply:
updated. Thank you :)

Reply:
Just this: `this.isEntityTypeFirm || this.isEntityTypeCP`

Reply:
Thanks for the cleanup!